timeseries/src/autogluon/timeseries/models/gluonts/abstract_gluonts.py

```diff
@@ -380,7 +368,7 @@ def _predict_gluonts_forecasts(
 
     @staticmethod
     def _sample_to_quantile_forecast(forecast: SampleForecast, quantile_levels: List[float]) -> QuantileForecast:
-        forecast_arrays = []
+        forecast_arrays = [forecast.mean]
```

Comment:
Why we need to add `mean` forecasts while the function only specify `quantile`?

Reply:
A `gluonts.model.forecast.QuantileForecast` object can store both the quantiles and the mean forecast (see `forecat_keys` [here](https://ts.gluon.ai/dev/api/gluonts/gluonts.model.forecast.html#gluonts.model.forecast.QuantileForecast)), so we save the mean here as well since we need to report it in output `TimeSeriesDataFrame`.